README.md

```diff
@@ -80,29 +80,28 @@ If you deploy your Lambda function as a zip file, you can add the extension
 to your Lambda layers using the console or [cli][lambda-add-layer-cli]:
 
 ```text
-arn:aws:lambda:<your-region>:634166935893:layer:vault-lambda-extension:8
+arn:aws:lambda:<your-region>:634166935893:layer:vault-lambda-extension:10
 ```
 
 #### 2. Option b) Install the extension for Lambda functions packaged in container images
 
 Alternatively, if you deploy your Lambda function as a container image, simply
-place the built binary in the `/opt/extensions` directory of your image. There
-is currently no publicly accessible download location for the pre-built binary.
+place the built binary in the `/opt/extensions` directory of your image.
 
-To build the binary from source:
+Fetch the binary from releases.hashicorp.com:
 
 ```bash
-# Requires Golang installed. Run from the root of this repository.
-GOOS=linux GOARCH=amd64 go build -o vault-lambda-extension main.go
+# Requires `curl` and `unzip`
+curl --silent https://releases.hashicorp.com/vault-lambda-extension/0.4.0/vault-lambda-extension_0.4.0_linux_amd64.zip \
+  --output vault-lambda-extension.zip
+unzip vault-lambda-extension.zip
 ```
```

Comment:
I wonder if it's worth mentioning the [SHA256SUMS file](https://releases.hashicorp.com/vault-lambda-extension/0.4.0/vault-lambda-extension_0.4.0_SHA256SUMS) as an integrity check for the download.

Reply:
Good idea, done.